build.xml

```diff
@@ -252,6 +253,9 @@
         <matches pattern="true" string="${build.isArmSoftFloat}"/>
       </and>
     </condition>
+    <condition property="jre.arch" value="mips64el">
+      <matches pattern="mips64el" string="${os.arch}"/>
+    </condition>
```

Comment:
This can be removed, in the following line the property `jre.arch` is set to `os.arch` and so no specialcase is needed here.